setup.py

```diff
@@ -1,18 +1,67 @@
+"""Setup the OTP java environment with maven."""
 import os
+import pathlib
 import zipfile
 from urllib.request import urlretrieve
 from subprocess import run
 
 
-MAVEN_PATH=os.path.join('bin', 'apache-maven-3.8.6', 'bin', 'mvn')
+BINARY_FOLDER = pathlib.Path("bin")
+OSM_NAME = "osmconvert64.exe"
+OSMCONVERT_DOWNLOAD_URL = "https://yadi.sk/d/Vnwc4kut3LCBFm"
+# TODO(MB) Find the most up to date version of maven by checking https://dlcdn.apache.org/maven/maven-3
+MAVEN_VERSION = "3.8.7"
```

Comment:
This needs to be set as 3.8.8 now. Until L12 TODO is implemented

Reply:
Someone fixed within the TRSE branch